modules/standard/Random.chpl

```diff
@@ -1045,15 +1045,24 @@ module Random {
         than creating a new stream for the purpose of the call.
 
         :arg arr: The array to be filled
-        :type arr: [] :type:`eltType`
+        :type arr: `[] T`
       */
-      proc fillRandom(arr: [] eltType) {
+      proc fillRandom(arr: []) {
         if(!arr.isRectangular()) then
           compilerError("fillRandom does not support non-rectangular arrays");
 
         forall (x, r) in zip(arr, iterate(arr.domain, arr.eltType)) do
           x = r;
       }
+      proc fillRandom(arr: [], min: arr.eltType, max:arr.eltType) {
```

Comment:
Missing a line break in between these functions.  I haven't verified, but I also think the built documentation will include this overload *after* the previous overload's documentation, when it would be more helpful to list it before if you intend it to cover both.

And since it will turn up in documentation, it would be good to call out the additional potential arguments

Reply:
As it is we cover the getNext with bounds after the no-bounds version. I'm hesitant to rearrange all of these for this PR but I will add documentation.

Reply:
(It is not obvious to me that covering the min/max version first is better in this case since the other version is simpler and easier to explain as a starting point; then the min/max is a fancy version / elaboration on it).

Reply:
It wouldn't have to be first, just before the main documentation if we don't feel like documenting both.  But documenting both is also good